cfme/infrastructure/provider/__init__.py

```diff
@@ -341,11 +320,13 @@ def step(self):
 
 @navigator.register(InfraProvider, 'Edit')
 class Edit(CFMENavigateStep):
+    VIEW = ProviderEditView
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        sel.check(Quadicon(self.obj.name, self.obj.quad_name).checkbox())
-        cfg_btn('Edit Selected Infrastructure Providers')
+        self.prerequisite_view.items.get_item(by_name=self.obj.name).check()
+        cfg = self.prerequisite_view.toolbar.configuration
```

Comment:
I think I'd rather see a line continuation here for the long call chain instead of a new variable